### DISCUSSION TITLE
Translating without weblate

### DISCUSSION THREAD

**Discussion by ghost** [libre-tube/LibreTube#4537] [General]
Sorry for opening a discussionhere, but I was wondering if it is possible to translate without using weblate. I have currently translated the entire strings_nl.xml file locally, becouse weblate is blocked in my educational environment.
Please let me know if I can submit a pull request for this.
I'd love to see the app in my native language. Thanks!

**Comment by Bnyro** [comment]
If it's an entirely new language, then merging a PR without using Weblate is not a problem. However, if you edited an existing language, it's possible that your changes conflict with the ones on Weblate - which could cause issues.
But if you already translated it, it's okay to open a PR for now.

**Reply by ghost** [reply]
Thanks - will open it.
Also thanks again for creating the app, works nicely and the design is next level 🔥 🔥

**Reply by ghost** [reply]
I just created the pull request. Thanks for your time.
